test/functional/finalization_state_restoration.py

```diff
@@ -14,14 +14,16 @@
     cleanup_datadir,
     connect_nodes,
     disconnect_nodes,
+    generate_block,
     initialize_datadir,
     sync_blocks,
     sync_mempools,
     wait_until,
 )
 
+
 def generate_block(proposer, count=1):
-    proposer.generatetoaddress(count, proposer.getnewaddress('', 'bech32'))
+    generate_block(proposer, count=count)
```

Comment:
I think we can remove the outer function

Reply:
yes, in case we need a default argument, we could change the signature of the imported function.